projects/1.16/assets/tensura-mod-that-time-i-got-reincarnated-as-a-slime/ttigraas/lang/zh_cn.json

```
@@ -0,0 +1,907 @@
+  "gamerule.randomPlunder.description": "设置为 “false”，可以在转生时从技能随机几率中移除掠夺/复制技能。",
+  "gamerule.randomRace": "随机种族",
+  "gamerule.randomRace.description": "设置转生后玩家是否可以选择种族。",
+  "gamerule.skillGriefing": "技能破坏",
+  "gamerule.skillGriefing.description": "设置某些技能是否可以更改或打破方块, 这可能会导致一些技能没有效果。",
+  "gamerule.soulAmount": "拥有魔王种进化魔王所需灵魂数量",
+  "gamerule.soulAmount.description": "拥有恶魔种时成为真正的魔王所需的灵魂数量",
+  "gamerule.spatialSpectator": "允许使用空间技能进入旁观模式",
+  "gamerule.spatialSpectator.description": "设置玩家是否可以使用某些空间技能进入旁观模式，如果关闭可能会导致某些技能无用。",
+  "gamerule.spawnDistance": "设置玩家转世时被传送的最大距离",
+  "gamerule.spawnDistance.description": "设置玩家转世时被传送的最大距离",
+  "item.minecraft.lingering_potion.effect.harvestfestival": "滞留型药水 魔王进化",
+  "item.minecraft.lingering_potion.effect.hellburnt": "滞留型药水 炎狱",
+  "item.minecraft.lingering_potion.effect.lionsheart": "滞留型药水 狮心",
+  "item.minecraft.lingering_potion.effect.paralyzing": "滞留型药水 麻痹",
+  "item.minecraft.lingering_potion.effect.reluctance": "滞留型 心无者-冷却",
+  "item.minecraft.lingering_potion.effect.virus": "滞留型药水 病毒",
+  "item.minecraft.potion.effect.harvestfestival": "药水 魔王进化",
+  "item.minecraft.potion.effect.hellburnt": "药水 炎狱",
+  "item.minecraft.potion.effect.lionsheart": "狮心",
+  "item.minecraft.potion.effect.paralyzing": "麻痹",
+  "item.minecraft.potion.effect.reluctance": "心无者",
+  "item.minecraft.potion.effect.virus": "药水 病毒",
+  "item.minecraft.splash_potion.effect.harvestfestival": "滞留型药水 魔王进化",
+  "item.minecraft.splash_potion.effect.hellburnt": "喷溅型药水 炎狱",
+  "item.minecraft.splash_potion.effect.lionsheart": "喷溅型 狮之心",
+  "item.minecraft.splash_potion.effect.paralyzing": "喷溅型药水 瘫痪",
+  "item.minecraft.splash_potion.effect.reluctance": "喷溅型 心无者 药水",
+  "item.minecraft.splash_potion.effect.virus": "喷溅型药水 病毒",
+  "item.minecraft.tipped_arrow.effect.harvestfestival": "药水箭 魔王进化",
+  "item.minecraft.tipped_arrow.effect.hellburnt": "药水箭 炎狱",
+  "item.minecraft.tipped_arrow.effect.lionsheart": "药水箭 狮子的心",
+  "item.minecraft.tipped_arrow.effect.paralyzing": "药水箭 麻痹",
+  "item.minecraft.tipped_arrow.effect.reluctance": "药水箭 心无者",
+  "item.minecraft.tipped_arrow.effect.virus": "药水箭 病毒",
+  "item.ttigraas.absolute_defense": "绝对防御",
+  "item.ttigraas.absolutedefense": "绝对防御",
+  "item.ttigraas.absorb_and_dissolve": "吸收并溶解",
+  "item.ttigraas.analyst": "解析者",
+  "item.ttigraas.anti_demon_mask_boots": "防魔面具靴子",
+  "item.ttigraas.anti_demon_mask_chestplate": "防魔面具胸甲",
+  "item.ttigraas.anti_demon_mask_helmet": "防魔面具",
+  "item.ttigraas.anti_demon_mask_leggings": "防魔面具护腿",
+  "item.ttigraas.antidemonmask_boots": "防魔面具靴子",
+  "item.ttigraas.antidemonmask_chestplate": "防魔面具胸甲",
+  "item.ttigraas.antidemonmask_helmet": "防魔面具",
+  "item.ttigraas.antidemonmask_leggings": "防魔面具护腿",
+  "item.ttigraas.antidemonmaskbody": "防魔面具胸甲",
+  "item.ttigraas.antidemonmaskboots": "防魔面具靴子",
+  "item.ttigraas.antidemonmaskhelmet": "防魔面具",
+  "item.ttigraas.antidemonmasklegs": "防魔面具护腿",
+  "item.ttigraas.armorsaurus": "生成 鳞甲蜥蜴",
```

```suggestion
  "item.ttigraas.armorsaurus": "鳞甲蜥蜴刷怪蛋",
```
如确认此处需要修改，请完整校对全文共103处关于`生成`的条目。